html-forms/index.html

```diff
@@ -4,72 +4,49 @@
     <meta charset="UTF-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <title>HTML Forms</title>
-    <style>
-      p.email {
-        margin: unset;
-      }
-      p.coding {
-        margin: unset;
-      }
-      p.comment {
-        margin: unset;
-      }
-      p.submit {
-        margin: unset;
-      }
-      div {
-        display: inline;
-      }
-    </style>
   </head>
   <body>
     <h1>Coding Form!</h1>
     <form action="/my-coding-form-page" method="post">
       <section>
-        <p class="email">Email</p>
-        <label for="email">
+        <label for="email">Email</label>
+        <div>
           <input
-            type="text"
+            type="email"
             id="email"
             placeholder="yourname@email.com"
             name="user_email" />
-        </label>
+        </div>
       </section>
 
       <section>
-        <p class="coding">Why is coding cool?</p>
-        <select id="coding" name="codingiscool">
-          <option value="selected">Select an option</option>
-          <option>Because it's interesting</option>
-          <option>Because it's fun</option>
-          <option>Because it's powerful</option>
-        </select>
+        <label for="whycoding?">Why is coding cool?</label>
+        <div>
+          <select id="coding" name="codingiscool">
```

Comment:
The `for` needs to either be "coding", or the `id` needs to be "whycoding". The `for `matches the `id`